### ISSUE TITLE
installfest: git instructions need to be clear...

### ISSUE THREAD

**Issue by jeanettehead** [denver-railsbridge/docs#4]
... about creating a folder for you website that will go "up" to github.

had issues with students "adding" their entire Desktop to github.

**Comment by jeanettehead** [comment]
https://jsbridge-installfest.herokuapp.com/javascript-installfest/create-a-github-pages-repository

**Comment by jeanettehead** [comment]
https://github.com/denver-railsbridge/docs/tree/javascript-installfest-docs/sites/en/javascript-installfest

**Comment by vuinguyen** [comment]
Also modify the Create a Webpage instructions to just name the page index.html, so in step 3 we don't have to do that mv command

**Comment by vuinguyen** [comment]
Hmm... so I don't know if I did the right thing, but I made updates to both the "create-a-github-pages-repository' and 'create a webpage' pages and uploaded a couple of new images (referenced in the github pages one). I wasn't sure if I was supposed to create a branch off of the javascript-installfest-docs branch or just use the javascript-installfest-docs branch. Long story short, I just used the javascript-installfest-docs branch and then pushed the changes up. Let me know if you have any questions about what I did, or any feedback on how I can handle the branches/pushing up code for the next issue I work on. Hope this helps!
